packages/angular/projects/tx-native-angular-sdk/src/lib/Services/translation.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { ReplaySubject } from 'rxjs';
+
+import { ITranslationServiceConfig } from '../interfaces';
+
+const { tx, t } = require('@transifex/native');
+
+/** Singleton Injection */
+@Injectable({
+  providedIn: 'root'
+})
+
+/**
+ * Service which wraps the Transifex Native library for using
+ * inside components
+ */
+export class TranslationService {
+  // Observable for detecting locale change
+  private _localeChanged = new ReplaySubject<string>(0);
+  localeChanged = this._localeChanged.asObservable();
+
+  // Observable for detecting translation change
+  private _contentTranslated = new ReplaySubject<string>(0);
+  contentTranslated = this._contentTranslated.asObservable();
+
+  constructor() { }
+
+  /**
+   * Initializes the translation service
+   * @param {ITranslationServiceConfig} config
+   * @returns void
+   */
+  public async init(config: ITranslationServiceConfig) {
+    tx.init({
+      token: config.token,
+      secret: config.secret,
+      sourceLocale: config.sourceLocale,
+    });
```

Comment:
Any reason to not just pass `tx.init(config)` here?

Reply:
Done